scripts/build-android.js

```diff
@@ -68,7 +68,7 @@ if (options.clean) {
 }
 
 if (!fs.existsSync(buildPath)) {
-  fs.mkdirSync(buildPath);
+  fs.mkdirSync(buildPath, { recursive: true });
 }
```

Comment:
Might as well go all the way and make use [`ensureDirSync`](https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/ensureDir-sync.md):

```
fs.ensureDirSync(buildPath);
```